test/shared/src/main/scala/zio/test/Assertion.scala

```diff
@@ -308,7 +313,7 @@ object Assertion extends AssertionVariants {
    */
   def isSubtype[A](assertion: Assertion[A])(implicit C: ClassTag[A]): Assertion[Any] =
     Assertion[Any](
-      SmartAssertions.as[Any, A].withCode(s"isSubtype") >>> assertion.arrow
+      SmartAssertions.as[Any, A].withCode("isSubtype", param(className(C))) >>> assertion.arrow
```

Comment:
Maybe for types we can stringify to `[<name>]` and so that would be another constructor:
```scala
final case class Type[a](classTag: ClassTag[a]) extends Argument
```